nibabel/testing/__init__.py

```diff
@@ -57,6 +59,21 @@ def assert_allclose_safely(a, b, match_nans=True):
     assert_true(np.allclose(a, b))
 
 
+def isiterable(iterable):
+    try:
+        for _ in iterable:
+            pass
+    except:
+        return False
+
+    return True
```

Comment:
I see. I didn't look deeply into how it was being used. Maybe something like `check_iteration`?